examples/structured_data/collaborative_filtering_movielens.py

```diff
@@ -0,0 +1,217 @@
+"""
+Title: Collaborative Filtering for Movie Recommendations
+Author: Siddhartha Banerjee
+Date created: 2020/05/24
+Last modified: 2020/05/24
+Description: Recommending movies using user and movie embeddings trained using Movielens small dataset.
+"""
+"""
+## Introduction
+
+This example looks at
+[Collaborative filtering using Movielens dataset](https://www.kaggle.com/c/movielens-100k)
+to recommend movies to users.
+"""
+
+import tensorflow as tf
+import pandas as pd
+import numpy as np
+from tensorflow import keras
+from pathlib import Path
+from tensorflow.keras import layers
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import LabelEncoder
+from tensorflow.keras.models import Model
+from tensorflow.keras.layers import (
+    Input,
+    Reshape,
+    Dot,
+    Add,
+    Embedding,
+    Activation,
+    Lambda,
+    Dense,
+)
+from tensorflow.keras.optimizers import Adam
+from tensorflow.keras.regularizers import l2
```

Comment:
For these 2 imports, just `from tensorflow import keras` and then use e.g. `keras.regularizers.l2`